src/schemas/json/foundryvtt-manifest.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "JSON schema for Foundry VTT system.json and module.json files.",
+  "type": "object",
+  "definitions": {
+    "stringArray": {
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "string"
+      }
+    }
+  },
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The unique package identifier. This must be lowercase and contain no special characters other than hyphens to separate multiple terms. This must also match the name of the directory."
+    },
+    "title": {
+      "type": "string",
+      "description": "The human readable title of the package which is displayed in the Setup menu and elsewhere."
+    },
+    "description": {
+      "type": "string",
+      "description": "A description of the package which appears in the Setup menu. May contain HTML tags to structure and style the text."
+    },
+    "version": {
+      "type": ["number", "string"],
+      "description": "Specifies the version number as a number or string. Incrementing this will allow Foundry to update the installed package for users and migrate their data if necessary."
+    },
+    "author": {
+      "type": "string",
+      "description": "The person who created the package. Alternatively you may use the `authors` property if multiple people have to be credited."
+    },
+    "authors": {
+      "type": "array",
+      "description": "Array of authors who have contributed to this package. Cannot be used together with `author`."
+    },
+    "minimumCoreVersion": {
+      "type": "string",
+      "description": "Specifies the minimum Foundry version required to be able to install and use this package."
+    },
+    "compatibleCoreVersion": {
+      "type": "string",
+      "description": "Specifies the latest Foundry version this package is compatible with."
+    },
+    "scripts": {
+      "$ref": "#definitions/stringArray",
+      "description": "A list of JavaScript file paths to include when this package is used. Paths are relative to the package root directory. Scripts will be loaded in the listed order as standard scripts."
+    },
+    "esmodules": {
+      "$ref": "#definitions/stringArray",
+      "description": "A list of JavaScript ES modules to include when this package is used. Paths are relative to the package root directory. Modules will be loaded in the listed order as modules."
+    },
+    "styles": {
+      "$ref": "#definitions/stringArray",
```

Comment:
Missing forward slash after #
"$ref": "#/definitions/stringArray"

Also the same issue with the other file

Reply:
Fixed, thank you!